soccer/gameplay/situational_play_selection.py

```diff
@@ -372,6 +372,7 @@ def updatePreempt(self):
             if (currentPlay != None and isinstance(
                     currentPlay, standard_play.StandardPlay) and self.enabled):
                 if (currentPlay.try_preempt()):
+                    print("Play has been preempted!")
```

Comment:
This should probably be a better message but I do think it should exist.

Reply:
Depending on what data you have, you should specify from what situation to what situation transition happened if it's not already there. It will be useful for debugging in the future if someone makes a play that triggers these transitions consistently